app/_hub/kong-inc/openid-connect/overview/_index.md

```diff
@@ -927,6 +929,24 @@ We can use the output in `Authorization` header.
    }
    ```
 2. Done.
+   
+#### Test the JWT Access Token Authentication with access token in query string 
+
+To specify the bearer token as a query string parameter:
+
+```bash
+curl -i -X PATCH http://localhost:8001/plugins/5f35b796-ced6-4c00-9b2a-90eef745f4f9  \
+  --data "config.bearer_token_param_type=query"                 \
+  --data "config.auth_methods=bearer"                           \
+  --data "config.auth_methods=password" # only enabled for demoing purposes
+```
+
+Test out the token by accessing the Kong proxy:
+```bash
+curl -i -X GET http://localhost:8000 \
+  -H "Authorization: Bearer <access-token>"
```

Comment:
1. The -H means it is sent as header and not query which is the example we wanted to show
   a. For query it is part of url - would be curl "http://localhost:8000?access_token=<token>"
2. The -i will only reply back with headers which should not be used for examples
3. Also It could be a GET or a POST or a PATCH. So no need to limit to GET. Curl can be shown without qualifying it. If there is no body, then it is GET. This part I leave to you based on the format of other example.

Reply:
> The -i will only reply back with headers which should not be used for examples

We actually almost always include `-i` in our cURL examples. It's helpful to make sure the user gets the right response - they can take it out if they want to, it doesn't hurt.